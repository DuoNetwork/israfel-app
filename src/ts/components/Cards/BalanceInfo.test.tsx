import { shallow } from 'enzyme';
import * as React from 'react';
import BalanceInfo from './BalanceInfo';

describe('BalanceInfo Test', () => {
	describe('Test Snapshot', () => {
		const icon = 'test';
		const name = '123';
		const value = 123;

		it('Test Snapshot', () => {
			const wrapper = shallow(
				<BalanceInfo
					icon={icon}
					name={name}
					value={value}
				/>
			);
			expect(wrapper).toMatchSnapshot();
			const wrapper1 = shallow(
				<BalanceInfo
					icon={icon}
					name={name}
					value={value}
				/>
			);
			expect(wrapper1).toMatchSnapshot();
			const wrapper2 = shallow(
				<BalanceInfo
					icon={icon}
					name={name}
					value={value}
				/>
			);
			expect(wrapper2).toMatchSnapshot();
		});
	});
});
