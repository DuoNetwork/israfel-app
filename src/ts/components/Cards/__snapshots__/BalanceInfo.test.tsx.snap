// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BalanceInfo Test Test Snapshot Test Snapshot 1`] = `
<styled.div>
  <div
    className="bg-logo"
  >
    <img
      src="test"
    />
  </div>
  <div
    className="tag-title"
  >
    <h3>
      test
    </h3>
  </div>
  <div
    className="tag-content"
  >
    <div>
      <div>
        <div
          className="tag-price"
        >
          12.00
        </div>
      </div>
    </div>
  </div>
</styled.div>
`;
